freyja/_cli.py

```diff
@@ -57,13 +70,43 @@ def print_barcode_version(ctx, param, value):
 @click.option('--depthcutoff', default=0,
               help='exclude sites with coverage depth below this value and'
               'group identical barcodes')
+@click.option('--lineageyml', default='-1', help='custom lineage.yml file')
```

Comment:
to make this a bit more clear, let's call it a "lineage hierarchy file (lineages.yml)"